src/main/java/uk/gov/digital/ho/hocs/queue/migration/MigrationCaseData.java

```diff
@@ -0,0 +1,106 @@
+package uk.gov.digital.ho.hocs.queue.migration;
+
+import com.jayway.jsonpath.JsonPath;
+import com.jayway.jsonpath.PathNotFoundException;
+import com.jayway.jsonpath.ReadContext;
+import lombok.extern.slf4j.Slf4j;
+import uk.gov.digital.ho.hocs.client.casework.dto.ComplaintCorrespondent;
+import uk.gov.digital.ho.hocs.document.JSONToSimpleTextConverter;
+import uk.gov.digital.ho.hocs.queue.complaints.ComplaintData;
+import uk.gov.digital.ho.hocs.queue.complaints.CorrespondentType;
+
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Slf4j
+public class MigrationCaseData implements ComplaintData {
+
+    public static final String CREATION_DATE = "$.creationDate";
```

Comment:
Do we require these to be public? Looks like most if not all are used within this class.

Reply:
Fixed